it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java

```diff
@@ -72,16 +67,16 @@ public Map<String, Object> submit(FormSubmitInfo formSubmitInfo) {
                         if (dataManager != null) {
                             String originalFileName = StringUtils.substringAfterLast(fileAttachment.getFileName(), "/");
                             String fileUrl = formSubmitInfo.getFormContainerPath() + "." + FileAttachmentServlet.SELECTOR + "/" + fileAttachmentUuid + "/" + originalFileName;
-                            fileAttachmentMapJsonObject = updateMapWithFileUrl(fileAttachmentMapJsonObject, fileAttachment.getFileName(), fileUrl);
-                            dataManager.put(fileAttachmentUuid, fileAttachment);
```

Comment:
does that means older custom prefill and submit service will not work with AF2 that used to work with AF1?

Reply:
@nit23uec The older was has been marked as old in cq guides [here](https://git.corp.adobe.com/livecycle/cq-guides/tree/master/it/aemds-guide-test-internal/src/main/java/com/adobe/aemds/guide/test/service)
The current one used in cq-guides is the implementation with a `List<FileAttachmentWrapper>` which now had to be used in core components as well and that's why it wasn't working.

Reply:
We don't support the deprecated file attachment map in the new servlets.